collect_app/src/main/AndroidManifest.xml

```diff
@@ -273,7 +273,6 @@
         <activity android:name="org.odk.collect.android.activities.CaptureSelfieVideoActivityNewApi" />
         <activity
             android:name="org.bcss.collect.naxa.login.LoginActivity"
-            android:noHistory="true"
```

Comment:
When I had made ```LoginActivity``` I never wanted it to in the backstack, This is the reason for noHistory flag being present here. 

This removal will make login activity to be present in the backstack(meaning when user logs in and then presses back button, app goes back to login screen), please either add finish to intent's starting from login activity or reintroduce the ```noHistory``` flag